Updates to most current Pages Get Started (#1665)
cloudflare/cloudflare-docs: products/pages/src/content/getting-started/index.md

# Getting started

In this guide, you will get started with Cloudflare Pages, and deploy your first website to the platform.

<Aside>

> You will be signing up for a Cloudflare account. You may also want a custom domain available, but it isn't required.

Reviewer: Should it also be `but it is not required` instead of `isn't`?

> You will be signing up for a Cloudflare account. You may also want a custom domain available, but it isn't required.

Author: ```suggestion
You will be signing up for a Cloudflare account. You may also want a custom domain available, but this is not required.
```

![Deploy log](./images/deploy-log.png)

When your project has finished deploying, you will receive a unique URL to view your deployed site.

## Managing your site

After your first deploy, select **Continue to project** to see see your project's configuration in the Cloudflare Pages dashboard. On this page, you can see your project's current deployment status, the production URL and associated commit, and all past deployments.

![Site dashboard](./images/site-dashboard.png)

### Adding a custom domain

While every Cloudflare Pages site receives a custom subdomain during deployment, you may also wish to point custom domains (or subdomains) to your site. To do this, go to **Custom domains** in your site dashboard.

![Domains](./images/domains.png)

 Select **Set up a domain**, provide the domain that you would like to serve your Cloudflare Pages site on and select **Continue**. Depending on the domain, you will need to configure nameservers or add a custom CNAME record to point the domain to your Cloudflare Pages site.

#### Configure nameservers

> Configure your nameservers to point to Cloudflare's name servers. If your nameservers are successfully pointed to Cloudflare, we will proceed by creating a CNAME record for you.

Author: ```suggestion
Configure your nameservers to point to Cloudflare's name servers. If your nameservers are successfully pointed to Cloudflare, Cloudflare will proceed by creating a CNAME record for you.
```

Reviewer 2: Nit: Looks odd using both "nameservers" and "name servers".

### Connect to GitHub

Signing in with GitHub allows [Cloudflare Pages](https://pages.dev) to deploy your projects, update your GitHub PRs with [preview deployments](/platform/preview-deployments), and more. After you sign in, select **Create a project**.

## Configuration and deployment

### Selecting a GitHub repository

You can select a GitHub project from your personal account or an organization you have given Pages access to. This allows you to choose a GitHub repository to deploy using Pages. Both private and public repositories are supported.

<Aside>

If you do not have a project, [refer to our extensive How To section](/how-to) to learn how to create and deploy common frameworks and tools to Cloudflare Pages.

</Aside>

### Configuring your deployment

Once you have selected a repository, select **Install & Authorize** and **Begin setup**. You can then customize your deployment in the "Set up builds and deployments" page.

Your "project name" will be used to generate your deploy's hostname. By default, this matches your GitHub project name.

> "Production branch" indicates the branch that Cloudflare Pages should use to deploy the "production" version of your site. For most projects, this is the `main` or `master` branch. All other branches that aren't your production branch will be used for [preview deployments](/platform/preview-deployments).

Reviewer 3: I'm somewhat amazed you let a contraction slip through 😅 

```suggestion
"Production branch" indicates the branch that Cloudflare Pages should use to deploy the "production" version of your site. For most projects, this is the `main` or `master` branch. All other branches that are not your production branch will be used for [preview deployments](/platform/preview-deployments).
```

Author: thank you for catching it!!

![Configuration](./images/configuration.png)

#### Build settings

Depending on the framework, tool, or project you are deploying to Cloudflare Pages, you will need to specify the site's **build command** and **publish directory** to tell Cloudflare Pages how to deploy your site.

> Our deployment UI provides a number of "presets", pre-filling values for many common deployment scenarios. If you are unsure what the correct values are for this section, check out our documentation for [build configuration](/platform/build-configuration).

Reviewer: I think replacing `,` for a `and` in `provides a number of "presets", pre-filling` will make it easier to understand.

> Our deployment UI provides a number of "presets", pre-filling values for many common deployment scenarios. If you are unsure what the correct values are for this section, check out our documentation for [build configuration](/platform/build-configuration).

Author: ```suggestion
Our deployment UI provides a number of "presets", pre-filling values for many common deployment scenarios. If you are unsure what the correct values are for this section, refer to the documentation for [build configuration](/platform/build-configuration).
```

![Build settings](./images/build-settings.png)

#### Environment variables

Environment variables are a common way of providing configuration to your build workflow. While setting up your project, you can specify a number of key-value pairs as environment variables. These can be further customized once your project has finished building for the first time.

After you have chosen your _Framework preset_, configured Root directory (optional), and customized your Environment variables (optional), you are ready to deploy. 

## Your first deploy

Once you have finished setting your build configuration, select **Save and Deploy**. Your project build logs will output as Cloudflare Pages installs your project dependencies, builds the project, and deploys it to Cloudflare's global network.

![Deploy log](./images/deploy-log.png)

When your project has finished deploying, you will receive a unique URL to view your deployed site. After this step, you have finalized deploying your first site with Cloudflare Pages.

## Managing your site

> After your first deploy, select **Continue to project** to see see your project's configuration in the Cloudflare Pages dashboard. On this page, you can see your project's current deployment status, the production URL and associated commit, and all past deployments.

Reviewer: Delete extra `see` in `select **Continue to project** to see see your project's`

Reviewer 4: I can't click on your block comment

> After your first deploy, select **Continue to project** to see see your project's configuration in the Cloudflare Pages dashboard. On this page, you can see your project's current deployment status, the production URL and associated commit, and all past deployments.

Author: ```suggestion
After your first deploy, select **Continue to project** to see your project's configuration in the Cloudflare Pages dashboard. On this page, you can see your project's current deployment status, the production URL and associated commit, and all past deployments.
```

![Site dashboard](./images/site-dashboard.png)

### Adding a custom domain

While every Cloudflare Pages site receives a custom subdomain during deployment, you may also wish to point custom domains (or subdomains) to your site. To do this, go to **Custom domains** in your site dashboard.

![Domains](./images/domains.png)

>  Select **Set up a domain**, provide the domain that you would like to serve your Cloudflare Pages site on and select **Continue**. Depending on the domain, you will need to configure nameservers or add a custom CNAME record to point the domain to your Cloudflare Pages site.

Reviewer: Change from `Select **Set up a domain**, provide` to "Select **Set up a domain** to provide...`
